### PR TITLE
Ftr: change timeQ from limit chan to ultimate chan

### PR DESCRIPTION
**Pull request by Lvnszn**
**What this PR does**:
change timeQ from limit chan to ultimate chan
**Which issue(s) this PR fixes**:

Fixes 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by justxuewei — APPROVED** [review]

File: time/timer.go [hunk 1]
```diff
@@ -570,6 +562,7 @@ func (w *TimerWheel) NewTimer(d time.Duration) *Timer {
 		t.w = timer.w
 		return t
 	}
+	log.Printf("[error] addTimer fail, err=%v\n", err)
```
Comment (AlexStocks) [inline time/timer.go:565]: use log.Errorf, pls.

**Review by LaurenceLiZhixin — APPROVED** [review]

**Comment by LaurenceLiZhixin** [comment]
@Lvnszn Please run go fmt.

File: time/timer.go (hunk 1 above)
Comment (Lvnszn) [reply time/timer.go]: PTAL

**Comment by Lvnszn** [comment]
> @Lvnszn Please run go fmt.

address

File: time/timer.go [hunk 2]
```diff
@@ -571,6 +562,7 @@ func (w *TimerWheel) NewTimer(d time.Duration) *Timer {
 		return t
 	}
 
+	gxlog.CError("addTimer fail, err is %v\n", err)
```
Comment (AlexStocks) [inline time/timer.go:565]: 我的理解，日志中不用加 "\n"

File: time/timer.go [hunk 3]
```diff
@@ -627,6 +619,7 @@ func (w *TimerWheel) NewTicker(d time.Duration) *Ticker {
 		return (*Ticker)(timer)
 	}
 
+	gxlog.CError("addTimer fail, err is %v\n", err)
```
Comment (AlexStocks) [inline time/timer.go:622]: 我的理解，日志中不用加 "\n"

File: time/timer.go [hunk 4]
```diff
@@ -637,6 +630,7 @@ func (w *TimerWheel) TickFunc(d time.Duration, f func()) *Ticker {
 	if err == nil {
 		return (*Ticker)(t)
 	}
+	gxlog.CError("addTimer fail, err is %v\n", err)
```
Comment (AlexStocks) [inline time/timer.go:633]: 我的理解，日志中不用加 "\n"

Comment (Lvnszn) [reply time/timer.go]: address, 我看了一下log函数，他里面写了换行

File: time/timer.go (hunk 2 above)
Comment (Lvnszn) [reply time/timer.go]: address, 我看了一下log函数，他里面写了换行

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**